app/src/lib/tokens/Result.svelte

```diff
@@ -14,34 +14,77 @@
 		{token.token}
 	</TokenDisplay>
 
-	<tr slot="header_row">
-		<th></th>
-		<th></th>
-		<th class="text-info-content">Level</th>
-		<th class="text-info-content">Gloss</th>
-	</tr>
+	<svelte:fragment slot="entry_row" let:entry>
+		{@const show_hints = entry.concept === null || [2, 3].includes(entry.concept.level)}
 
-	<tr slot="entry_row" let:entry>
 		{#if entry.concept}
 			{@const concept = entry.concept}
 			{@const term = concept_with_sense(concept)}
-			<td class="whitespace-nowrap">
-				<a class="link not-prose {classes}" href={`${PUBLIC_ONTOLOGY_API_HOST}/?q=${term}`} target="_blank">
-					{term}
-				</a>
-			</td>
-			<td class="whitespace-nowrap">{concept.part_of_speech}</td>
-			<td class="whitespace-nowrap">
-				<span class="badge badge-md L{concept.level} font-mono">L{concept.level}</span>
-			</td>
-			<td>{concept.gloss}</td>
-			<!-- TODO integrate hints from how-to -->
+			<tr>
+				<td class="whitespace-nowrap">
+					<a class="link not-prose {classes}" href={`${PUBLIC_ONTOLOGY_API_HOST}/?q=${term}`} target="_blank">
+						{term}
+					</a>
+				</td>
+				<td class="whitespace-nowrap">{concept.part_of_speech}</td>
+				<td class="whitespace-nowrap">
+					<span class="badge badge-md L{concept.level} font-mono">L{concept.level}</span>
+				</td>
+				<td>{concept.gloss}</td>
+			</tr>
 		{:else}
-			<td class="whitespace-nowrap">{entry.stem}</td>
-			<td class="whitespace-nowrap">{entry.part_of_speech}</td>
-			<td>N/A</td>
-			<!-- TODO integrate hints from how-to -->
-			<td>Not in Ontology. Check the How-To document.</td>
+			<tr>
+				<td class="whitespace-nowrap">{entry.stem}</td>
+				<td class="whitespace-nowrap">{entry.part_of_speech}</td>
+				<td>N/A</td>
+				<td>Not in Ontology.</td>
+			</tr>
 		{/if}
-	</tr>
+		
+		{#if show_hints}
+			{@const has_structure = entry.how_to.some(hint => hint.structure)}
+			{@const has_pairing = entry.how_to.some(hint => hint.pairing)}
+			{@const has_explication = entry.how_to.some(hint => hint.explication)}
+			{@const has_any_hint = has_structure || has_pairing || has_explication}
+
+			<tr class="border-none">
+				<td></td>
+				<td colspan="3" class="pb-2" class:ps-0={has_any_hint}>
+					{#if has_any_hint}
+						<table class="table table-xs my-0 border-none">
```

Comment:
I could refactor to make a `Table.svelte` component if you want, which we could use in `NotFound.svelte` and twice in this file. We'd have to rework the `PopupTable` component too.

Reply:
Nevermind, I did it